src/portal_visualization/builders/anndata_builders.py

```diff
@@ -18,14 +17,14 @@ class RNASeqAnnDataZarrViewConfBuilder(ViewConfBuilder):
     https://portal.hubmapconsortium.org/browse/dataset/e65175561b4b17da5352e3837aa0e497
     """
 
-    def __init__(self, entity, groups_token, assets_endpoint, **kwargs):
-        super().__init__(entity, groups_token, assets_endpoint, **kwargs)
+    def __init__(self, entity, groups_token, assets_endpoint):
+        super().__init__(entity, groups_token, assets_endpoint)
```

Comment:
The `**kwargs` has masked bugs: We thought we were using the right kwarg, but there was a typo, so the code assumed no value was supplied and it continued on without complaint.